### PR TITLE
3.5.33 release notes

### PR DESCRIPTION
**Pull request by brianhlin**

### CODE REVIEW

File: docs/release/3.5/release-3-5-33.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+title: OSG 3.5.33
+
+OSG Software Release 3.5.33
+===========================
+
+**Release Date:** 2021-04-01  
+**Supported OS Versions:** EL7, EL*
```
Comment (matyasselmeci) [inline docs/release/3.5/release-3-5-33.md:7]: This should either be "EL7, EL8" or "EL7"

File: docs/release/3.5/release-3-5-33.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    -   [Incompatibility with the multi-user plugin](https://github.com/opensciencegrid/xrootd-multiuser/issues/21):
+        users of the XRootD multi-user plugin will be unable to update to XRootD 5.1.x until a fixed version
+        of XRootD multi-user is released into the OSG repositories
+    -   In some cases, XCaches using the Rucio plug-in may crash due to malformed URLs generated by the plug-in.
+
+This release contains the following changes to the `osg-upcoming` repository:
+
+-   `XRootD 5.1.1` (EL7 only): an update from the previously available version, XRootD 5.0.2
+    -  See [this section](../updating-to-osg-35.md#updating-to-xrootd-5) for detailed update instructions
+    -  The XRootD SciTokens plug-in has been merged into the XRootD code-base so its version now matches the `xrootd`
```
Comment (matyasselmeci) [inline docs/release/3.5/release-3-5-33.md:37]:
```suggestion
    -  The XRootD SciTokens plug-in (`xrootd-scitokens`) has been merged into the XRootD code-base so its version now matches the `xrootd`
```
If you're going to use the RPM name for XRootD, you should use it for the plugin too.

File: docs/release/3.5/release-3-5-33.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+       ([SOFTWARE-4431](https://opensciencegrid.atlassian.net/browse/SOFTWARE-4431))
+    -  Added requirement for XRootD 5.1
+       ([SOFTWARE-4431](https://opensciencegrid.atlassian.net/browse/SOFTWARE-4431))
+-   `xrootd-hdfs 2.2.0`
+    -   Fixed a bug where checksums were written on HDFS write failures, potentially confusing clients
+        ([#34](https://github.com/opensciencegrid/xrootd-hdfs/pull/34))
+    -   Added support for new extended attribute API, allowing token plugins to properly set the username needed by HDFS
+        ([#35](https://github.com/opensciencegrid/xrootd-hdfs/pull/35))
+-   `xrootd-cmstfc 1.5.2-6`: rebuild against XRootD 5.1.1
+-   `xrootd-rucioN2N-for-Xcache 1.2-3.3`: rebuild against XRootD 5.1.1
```
Comment (matyasselmeci) [inline docs/release/3.5/release-3-5-33.md:52]:
```suggestion
-   `xrootd-cmstfc 1.5.2-6`: rebuilt against XRootD 5.1.1
-   `xrootd-rucioN2N-for-Xcache 1.2-3.3`: rebuilt against XRootD 5.1.1
```

File: docs/release/3.5/release-3-5-33.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+This release contains the following changes to the `osg-upcoming` repository:
+
+-   `XRootD 5.1.1` (EL7 only): an update from the previously available version, XRootD 5.0.2
+    -  See [this section](../updating-to-osg-35.md#updating-to-xrootd-5) for detailed update instructions
+    -  The XRootD SciTokens plug-in has been merged into the XRootD code-base so its version now matches the `xrootd`
+       package version
+    -  See the [upstream release notes](https://github.com/xrootd/xrootd/blob/v5.1.1/docs/ReleaseNotes.txt#L9-L164) for
+       details
+-   `XCache 2.0.0`
```
Comment (matyasselmeci) [inline docs/release/3.5/release-3-5-33.md:41]:
```suggestion
-   XCache 2.0.0
```
just a style thing -- if you're using a software name, don't use monospace; if you're using a package name, use monospace

Comment (brianhlin) [reply docs/release/3.5/release-3-5-33.md]: Fixed, bolded instead

**Review by matyasselmeci — APPROVED** [review]

**Pull request merged**